website/docs/docs/collaborate/model-query-history.md

```diff
@@ -9,7 +9,7 @@ image: /img/docs/collaborate/dbt-explorer/model-query-queried-models.jpg
 
 The model query history tile allows you to:
 
-- View the query count for a model based on the data warehouse's query logs.
+- View the count of consumption queries for a model based on the data warehouse's query logs.
 - Provides data teams insight, so they can focus their time and infrastructure spend on the worthwhile used data products.
 - Enable analysts to find the most popular models used by other people.
 
```

Comment:
should we have a 'what is a consumption query' section?
```suggestion
<Expandable alt_header="What's a consumption query"?>

A consumption query is the number of queries in your dbt project that has used the model in a given time. 

For example, if a model is used in multiple `select` queries across your organization, each query is counted as a consumption query. This allows teams to measure how often a particular model is being used by data practitioners or other tools, giving insight into the value and frequency of its use. 

So if `model_super_santi` was queried 10 times in the past week, it would count as having 10 consumption queries for that particular time period.
</Expandable>

```

Reply:
Consumption query is a metric...